.github/workflows/continuous-integration.yml

```diff
@@ -57,5 +57,8 @@ jobs:
     - name: Release Heroku container
       run: heroku container:release web --app heroku-app-devops
```

Comment:
You could delete the Heroku steps instead of maintaining two deployments of your app. But not a problem if you want to keep it.